docs/readmes/README.managed-mode.md

```diff
@@ -0,0 +1,41 @@
+# Pod Identity in Managed mode
+> Available from 1.6.0 release
+
+## Introduction
+
+Starting from 1.6.0 release,2 modes of operation are supported for pod-identity
+- Standard Mode
+- Managed Mode
+
+### Standard Mode
+
+This is the default mode in which pod-identity will be deployed. In this mode, there are 2 components, MIC (Managed Identity Controller) and NMI (Node Managed Identity). MIC handles the identity assignment/removal from the underlying vm/vmss when new pods using the identity are created/deleted.
+
+### Managed Mode
+
+In this mode, there is only the NMI component deployed in the cluster. The identity assignment needs to be manually performed.
```

Comment:
Let's mention here that `forceNamespaced` is enabled by default for managed mode.